src/main/scala/generate/database/PostgresGenerator.scala

```diff
@@ -29,8 +29,13 @@ object PostgresGenerator extends DatabaseGenerator {
         sb.append(s"CREATE TABLE $tableName ")
         val stringColumns = columns map parseColumn
         sb.append(stringColumns.mkString("(\n", ",\n", "\n)"))
-        sb.append(";\n")
+      case Read(columns, tableName) =>
+        sb.append("SELECT")
+        val stringColumns = columns map parseColumn
+        sb.append(stringColumns.mkString("\n", ",\n", "\n"))
```

Comment:
Personally I'd go with the following, rather than lots of newlines?
```scala
sb.append("SELECT ")
val stringColumns = columns map parseColumn
sb.append(stringColumns.mkString(", "))
sb.append(s" FROM ${indent(tableName)}")
```
Thoughts, @jaylees14, @Lewiky?